react/src/Attachment/utils/getErrorMessage.ts

```diff
@@ -0,0 +1,22 @@
+import { ErrorCode, FileRejection } from 'react-dropzone'
+
+import { getFileExtension } from './getFileExtension'
+
+export const getErrorMessage = (rejectedFile: FileRejection): string => {
+  const firstError = rejectedFile.errors[0]
+  let errorMessage
+  switch (firstError.code) {
+    case ErrorCode.FileInvalidType: {
+      const fileExt = getFileExtension(rejectedFile.file.name)
+      errorMessage = `Your file's extension ending in *${fileExt} is not allowed`
```

Comment:
nit: would it be easier to just return directly in the switch case instead of setting a variable?

```suggestion
      return `Your file's extension ending in *${fileExt} is not allowed`
```

same for the rest of this switch statement